lib/Tool/Authentication.php

```diff
@@ -54,7 +54,11 @@ public static function authenticateSession(?Request $request = null): ?User
 
             if ($user instanceof \Pimcore\Security\User\User && self::isValidUser($user->getUser())) {
                 $pimcoreUser = $user->getUser();
-                $pimcoreUser->setLastLoginDate(); //set user current login date
+                // update last login date if last login timestamp was more than few seconds ago
```

Comment:
this is to avoid to re-update the same value or updates in some sub-requests that happens few seconds apart, as we are not needing a real-time value